File: csharp/MaxPainUI/ClientApp/src/app/models/blog-entry.ts
export class BlogEntry {
    id: number;
    title: string;
    imageUrl: string;
    content: string;
    ordinal: number;
    isActive: boolean;
    isStockPick: boolean;
    showOnHome: boolean;
    createdOn: Date;
    modifiedOn: Date;

    constructor() { }

    get dash(): string {
        return this.title.replace(/\s/g, '-');
    }

    get summary(): string {
        return this.content.substr(30) + "...";
    }
} 

File: csharp/MaxPainInfrastructure/Models/BlogEntry.cs
namespace MaxPainInfrastructure.Models
{
    public class BlogEntry
    {
        public long Id { get; set; }
        public string? Title { get; set; }
        public string? ImageUrl { get; set; }
        public int Ordinal { get; set; }
        public bool IsActive { get; set; }
        public bool IsStockPick { get; set; }
        public bool ShowOnHome { get; set; }
        public DateTime? CreatedOn { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public string? Content { get; set; }
    }
}
